fuel-indexer-schema/src/db.rs

```diff
@@ -71,6 +76,33 @@ impl IndexerConnectionPool {
         }
     }
 
+    pub async fn is_connected(&self) -> sqlx::Result<ServiceStatus> {
```

Comment:
I tried [`pg_isready`](https://www.postgresql.org/docs/current/app-pg-isready.html) as well but thought this was a little more idiomatic (as opposed to using `Command`)